bin/exec.ml

```diff
@@ -72,9 +72,7 @@ module Watch = struct
 
   let kill_process pid =
     let pid_int = Pid.to_int pid in
-    (* TODO This logic should exist in one place. Currently it's here and in
```

Comment:
cc @gridbugs I am not sure if this TODO was referring to the line below, but I could not find exactly this same logic in the `Scheduler` module.